codecov.yml

```diff
@@ -1,6 +1,3 @@
-codecov:
```

Comment:
why this? should we isolate this in another PR?

Reply:
ok, I was trying to experiment what we discussed in triage codecov channel, I'll revert this one.

Reply:
ah well, if it is for investigating is good, does it make any improvement?

Reply:
@JoanFM I didn't see, because python 3.9 ci failed and... coverage report not being uploaded